### PR TITLE
fix(groups): Improve special comment keywords

### PR DESCRIPTION
**Pull request by cryptomilk**
This improve special comment keywords.

Here is how it will look like:

gruvbox-comments

### CODE REVIEW

File: lua/gruvbox/groups.lua [hunk 1]
```diff
@@ -118,7 +118,9 @@ M.setup = function()
     lCursor = { link = "Cursor" },
     Special = { link = "GruvboxOrange" },
     Comment = { fg = colors.gray, italic = config.italic.comments },
-    Todo = { fg = colors.fg0, bold = config.bold, italic = config.italic.comments },
+    -- Todo         anything that needs extra attention; mostly the
+    --              keywords TODO FIXME and XXX
+    Todo = { fg = colors.bg0, bg = colors.yellow, italic = config.italic.comments },
```
Comment (ellisonleao) [inline lua/gruvbox/groups.lua:123]: can you keep the bold config please?

File: lua/gruvbox/groups.lua [hunk 2]
```diff
@@ -362,10 +364,14 @@ M.setup = function()
     ["@text.todo"] = { link = "Todo" },
     -- @text.note             ; info notes
     ["@text.note"] = { link = "SpecialComment" },
+    -- @text.note.comment     ; XXX in comments
+    ["@text.note.comment"] = { fg = colors.purple },
```
Comment (ellisonleao) [inline lua/gruvbox/groups.lua:368]: can you add the bold config here too?

File: lua/gruvbox/groups.lua [hunk 3]
```diff
@@ -362,10 +364,14 @@ M.setup = function()
...
     -- @text.warning          ; warning notes
     ["@text.warning"] = { link = "WarningMsg" },
     -- @text.danger           ; danger/error notes
     ["@text.danger"] = { link = "ErrorMsg" },
+    -- @text.danger.comment   ; FIXME in comments
+    ["@text.danger.comment"] = { fg = colors.fg0, bg = colors.red },
```
Comment (ellisonleao) [inline lua/gruvbox/groups.lua:374]: can you add the bold config here too?

**Review by ellisonleao — CHANGES_REQUESTED** [review]

File: lua/gruvbox/groups.lua (hunk 1 above)
Comment (cryptomilk) [reply lua/gruvbox/groups.lua]: Done.

**Comment by willeccles** [comment]
Is there a convenient way to undo this in my config? I can't find a clean way to have the equivalent of `fg = colors.fg0` in `overrides` and I do not like the new highlighting.

**Pull request merged**